features/karaf/src/main/feature/feature.xml

```diff
@@ -12,8 +12,8 @@
 <features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
 
     <feature name="openhab-runtime-base" description="openHAB Runtime Base" version="${project.version}">
-        <feature>shk-esh-base</feature>
-        <feature>shk-esh-io-console-karaf</feature>
+        <feature>esh-base</feature>
```

Comment:
I left it out as I was thinking on reusing the openhab-runtime-base for the minimal distro, but we'll probably need to do something special there anyway.
I will add the features as you suggested above. Any others that should be included?